src/DependencyInjection/ConfigResolver/HttpCacheConfigParser.php

```diff
@@ -33,6 +33,14 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
                         ->example(array('http://localhost/', 'http://another.server/'))
                         ->requiresAtLeastOneElement()
                         ->prototype('scalar')->end()
+                    ->end()
+                    ->scalarNode('purge_auth_header')
+                        ->info('Header name for authenticated purge')
+                        ->defaultValue('X-PURGE-AUTH')
+                    ->end()
+                    ->scalarNode('purge_auth_key')
+                        ->info('Purge authentication key')
```

Comment:
Besides mentioning it is for Varnish, also make it clear it's optional as normally auth is done using ACL (ip based).